src/openforms/js/components/utils.js

```diff
@@ -0,0 +1,9 @@
+import FormIOModule from 'formio_module';
+
+const getComponentEmptyValue = component => {
+  const componentClasses = FormIOModule.components;
```

Comment:
I'd consider looking into formio's component builder itself instead of only our custom module:

* not all component types are registered in our module
* the module only specifies overrides/custom components, formio will fall back to its own components

```js
import Components from 'formio/components/Components';

const instance = Components.create(component);
```

See: https://github.com/formio/formio.js/blob/master/src/components/Components.js#L59